examples/lock-app/efr32/include/EventHandler.h

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#if defined(ENABLE_CHIP_SHELL)
```

Comment:
The `#include` already has a if defined check. 
What are the rules here? I think only one place for ifdefs should be used.

Reply:
Yeah I think this one can be removed if the #include is in the ifdef